lms/djangoapps/certificates/api.py

```diff
@@ -40,8 +41,10 @@
     certificate_status_for_student
 )
 from lms.djangoapps.certificates.queue import XQueueCertInterface
-from lms.djangoapps.certificates.utils import get_certificate_url as _get_certificate_url
-from lms.djangoapps.certificates.utils import has_html_certificates_enabled as _has_html_certificates_enabled
+from lms.djangoapps.certificates.utils import (
+    get_certificate_url as _get_certificate_url,
+    has_html_certificates_enabled as _has_html_certificates_enabled
```

Comment:
Are we importing these with an underscored name to highlight that they are internal methods?

Reply:
Yep